### ISSUE TITLE
Manifest merger failed when add `android:supportsRtl="false"` to AndroidManifest.xml

### ISSUE THREAD

**Issue by keima** [kobakei/spot#6]
when add `android:supportsRtl="false"` to AndroidManifest.xml,
show Gradle Build message:

```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@supportsRtl value=(false) from AndroidManifest.xml:11:9-36
  	is also present at [com.github.kobakei.spot:library:0.1.3] AndroidManifest.xml:14:9-35 value=(true).
  	Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:5:5-23:19 to override.
```

Workaround is suggested, so I bypass this issue.
But probably this issue occurs that defined `android:supportsRtl="true"` to Spot's AndroidManifest.
https://github.com/kobakei/spot/blob/master/library/src/main/AndroidManifest.xml

I think this definition (and `android:allowBackup="true"`) is unnecessary for spot.
So please remove this.

**Comment by kobakei** [comment]
Thank you for your feedback.
The issue has been fixed at 0.1.4.
Please try it!

**Comment by keima** [comment]
Thanks!
